scripts/deploy.ts

```diff
@@ -110,6 +178,90 @@ async function main() {
   )
 }
 
+const legacyContractAddress = '0x9e7A15E77e5E4f536b8215aaF778e786005D0f8d'
+
+async function downloadData(
+  factory: OBSSStorage__factory,
+  provider: JsonRpcProvider,
+  signer: Signer
+) {
+  const legacyContract = factory
+    .attach(legacyContractAddress)
+    .connect(provider)
+    .connect(signer)
+
+  console.log(legacyContract.address)
+  const totalFeeds = await legacyContract.lastFeedId()
+  console.log(`Total feeds count: ${totalFeeds.toNumber()}`)
+  const legacyPosts: OBSSStorage.LegacyPostStruct[] = []
+  const legacyReactions: OBSSStorage.LegacyReactionStruct[] = []
+  for (let i = 0; i < 10; i++) {
```

Comment:
Why `i < 10`? What is `10`?

Reply:
Test value, it's going to be `totalFeeds`